src/api/V1/Fonts/Api_Fonts.php

```diff
@@ -124,15 +124,30 @@ public function register_endpoint() {
 			'/fonts/',
 			[
 				'methods'  => \WP_REST_Server::READABLE,
-				'callback' => [ $this, 'response' ],
-
+				'callback' => [ 'save_font' ],
```

Comment:
This should be `[ $this, 'save_font' ]`, as it uses PHP Callbacks functionality under the hood https://www.php.net/manual/en/language.types.callable.php